app/views/dashboard/_polling.html.erb

```diff
@@ -1,3 +1,5 @@
 <script>
-  MinionPoller.poll();
+  document.addEventListener("DOMContentLoaded", function(event) { 
```

Comment:
Yes it is but it is not loaded at this point. There are ways to fix this but I didn't want to introduce any Javascript loading logic or take any decisions now that will have to be changed later. Vanilla Javascript to the rescue...